unstyled_docs/spec/intro.html

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+  <body>
+    <!-- redirect with javascript if possible, to preserve the fragment. -->
+    <script>
```

Comment:
if you're insisting on writing xhtml 1(!) i think this has to have a `type="text/javascript"` technically.

Reply:
in fact, it's worse - for xhtml 1 you'd have had to have done

```
<script type="text/javascript">
//<![CDATA[
...
//]]>
</script>
```

to be safe. surely just use html5 (i.e. dump the XML header, and have a DTD of `<!DOCTYPE html>` and then you can just use a nude `<script>` tag.

Reply:
ugh I hate this [x]html[12345] stuff. I just cargoculted the headers from somewhere else. will update